chains/solana/contracts/programs/ccip-router/src/state.rs

```diff
@@ -67,8 +69,49 @@ pub struct GlobalState {
 #[derive(Clone, AnchorSerialize, AnchorDeserialize, InitSpace, Debug)]
 pub struct SourceChainConfig {
     pub is_enabled: bool, // Flag whether the source chain is enabled or not
-    #[max_len(64)]
-    pub on_ramp: Vec<u8>, // OnRamp address on the source chain
+    #[max_len(128)]
+    pub on_ramp: Vec<u8>, // OnRamp addresses supported from the source chain, each of them has a 64 byte address. So this can hold 2 addresses.
```

Comment:
Any reason no to make this a `Vec<Vec<u8>>` instead? Then you could set `max_len(2,64)` on it. Or maybe even `[[u8;64];2]`, though I know the latter sometimes hits complications with anchor